src/EventService.Data/EventRepository.cs

```diff
@@ -39,6 +41,11 @@ public EventRepository(IDataProvider provider)
   public Task<bool> DoesExistAsync(Guid eventId)
   {
     return _provider.Events.AnyAsync(e => e.Id == eventId && e.IsActive);
+  }
+
+  public async Task<List<Guid>> DoExistAsync(List<Guid> eventsIds)

```

Comment:
name doesn't match action. GetExisting would be better

Reply:
done